src/Oscoin/Telemetry.hs

```diff
@@ -82,6 +82,12 @@ emit Handle{..} evt = withLogger $ GHC.withFrozenCallStack $ do
         BlockMinedEvent blockHash ->
             Log.withNamespace "node" $
                 Log.infoM "mined block" fmtBlockHash blockHash
+        BlockchainTipExtended blockHash ->
```

Comment:
Not necessarily now, but I think we should start nesting related event types, and have fewer `NotableEvent` constructors

Reply:
Good idea -- perhaps we could factor them out into "semantic buckets" i.e. `gossip` , `storage`, etc?